webvtt/rendering/cues-with-video/processing-model/basic-ref.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <title>Reference for WebVTT rendering, basic</title>
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
```

Comment:
For the 3 WebVTT tests, it appears that they were failing in Chrome before and still are, but in a different way. It looks like Ahem is loaded so it doesn't suggests a problem with these changes. Landing these changing without understanding the differences wouldn't be so bad, since the tests already aren't providing any regression detection value in Chromium.

My best guess for what's changed is that some font metrics are computed differently for web fonts and system fonts. @drott would be able to confirm if this is the case, and it could explain lots of differences.

Reply:
Ahh, sorry, didn't realize these were already failing. They popped up in the wpt.fyi check in #17173 and I just blindly moved them to here. It's strange that they appeared there, though, and the link is gone. But makes sense to just submit these.